config/default/samlauth.authentication.yml

```diff
@@ -1,32 +1,64 @@
+metadata_cache_http: false
+local_login_saml_error: false
 security_authn_requests_sign: true
-security_messages_sign: true
+security_logout_requests_sign: true
+security_logout_responses_sign: true
 strict: true
-drupal_saml_login: false
+security_messages_sign: true
+security_lowercase_url_encoding: true
+request_set_name_id_policy: true
+security_want_name_id: true
+security_request_authn_context: true
+use_base_url: true
+_core:
+  default_config_hash: oDGEkhP0h5rXXqlDplxeBDre0goLigOJupHKMDMwcqM
+login_menu_item_title: ''
+logout_menu_item_title: ''
+logout_different_user: false
+login_redirect_url: ''
+logout_redirect_url: ''
+drupal_login_roles:
+  authenticated: '0'
+  viewer: '0'
+  editor: '0'
+  publisher: '0'
+  webmaster: '0'
+  administrator: '0'
+error_redirect_url: ''
+error_throw: false
 sp_entity_id: ''
 sp_name_id_format: 'urn:oasis:names:tc:SAML:2.0:nameid-format:transient'
-sp_x509_certificate: ' '
-sp_private_key: ' '
+sp_x509_certificate: Overridden
+sp_private_key: Overridden
 sp_cert_folder: ''
 idp_entity_id: ''
 idp_single_sign_on_service: ''
 idp_single_log_out_service: ''
 idp_change_password_service: 'https://apps.its.uiowa.edu/pwtool/change'
+idp_cert_type: single
 idp_x509_certificate: ''
+idp_x509_certificate_multi: ''
 unique_id_attribute: 'urn:oid:1.3.6.1.4.1.6159.1.41'
 map_users: true
+map_users_name: true
+map_users_mail: true
+map_users_roles:
```

Comment:
This is not great for splits. Sites with role splits will either need to split this config out or we can dynamically override it in uiowa_auth.

Reply:
Done